Makefile

```diff
@@ -145,14 +145,14 @@ generate: controller-gen ## Generate code containing DeepCopy, DeepCopyInto, and
 	$(CONTROLLER_GEN) object:headerFile="hack/boilerplate.go.txt" paths="./..."
 
 # Path used to import Gatekeeper manifests
-IMPORT_MANIFESTS_PATH ?= https://github.com/open-policy-agent/gatekeeper
+IMPORT_MANIFESTS_PATH ?= https://github.com/stolostron/gatekeeper
 TMP_IMPORT_MANIFESTS_PATH := $(shell mktemp -d)
 GATEKEEPER_MANIFEST_DIR ?= config/gatekeeper
 
 .PHONY: import-manifests
 import-manifests: kustomize ## Import Gatekeeper manifests.
 	if [ "$${IMPORT_MANIFESTS_PATH#https://}" != "$(IMPORT_MANIFESTS_PATH)" ]; then \
-		git clone --depth=1 --branch v$(GATEKEEPER_VERSION)  $(IMPORT_MANIFESTS_PATH) $(TMP_IMPORT_MANIFESTS_PATH) ; \
+		git clone --depth=1 --branch release-$(shell echo $(VERSION) | cut -d '.' -f 1-2 )  $(IMPORT_MANIFESTS_PATH) $(TMP_IMPORT_MANIFESTS_PATH) ; \
```

Comment:
Why was this changed? I think we still want to use the tag here rather than just the latest commit?

Reply:
because I wanted to clone from `stolostron/gatekeeper` branch `release-3.X `

Reply:
Do you want tag?

Reply:
@dhaiducek Dale I changed to `release-$(shell echo $(VERSION)` again because CI failed when a tag does not include our own commits(stolostron commits)

Reply:
Okay. And actually, thinking about it, we probably should be just adding a `-#` suffix to tags in `stolostron/gatekeeper` rather than deleting and recreating the same tag. I'm thinking let's merge it this way and then update the `build-push-stolostron` workflow to remove the trailing suffix so that it's overwriting the image tag each time. What do you think of that?

Reply:
Quay gatekeeper's tag going to be v3.X.X-X like this?